From 5cd7df786d26e6a9797e6d8373ce13da388873af Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Fri, 19 Jan 2024 21:04:11 -0500
Subject: [PATCH] Use normal numpy as build dependency

We don't need NumPy 2, as we don't need to support multiple NumPy
versions.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 23c2751b..9f870644 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -53,7 +53,7 @@ ducc = ['ducc0>=0.15']
 build-backend = 'mesonpy'
 requires = [
   'meson-python>=0.14',
-  'numpy>=2.0.0',
+  'numpy',
   'jupyter-core',
   'setuptools>=64',
   'setuptools_scm>=8',
-- 
2.45.2

